content/docs/contributing/release-process.md

```diff
@@ -385,7 +385,7 @@ page if a step is missing or if it is outdated.
       > kicking off a build using the steps in `gcb/build_cert_manager.yaml`. Users with access to
       > the cert-manager-release project on GCP should be able to view logs in [GCB build history](https://console.cloud.google.com/cloud-build/builds?project=cert-manager-release).
 
-9. **(1.12 and above)** In this step, we make sure the Go module
+9. **(1.12, 1.13, and 1.14)** In this step, we make sure the Go module
```

Comment:
Is there maybe a way to collapse this item, so it becomes harder to run these steps by accident?

Reply:
I implemented this in https://github.com/cert-manager/website/pull/1456/commits/d83ebca4c4606124e939547227d7bf279b9d1b6d PTAL.

Reply:
LGTM